frontend/packages/sdk/README.md

```diff
@@ -89,10 +89,26 @@ const options = {
   gasLimit: gasRequired.refTime,
   storageDepositLimit: storageDeposit.isCharge ? storageDeposit.asCharge : null,
 }
-await contract.tx.inc(options, incValue).signAndSend(pair, { nonce: -1 })
+const result = await contract.tx.inc(options, incValue).signAndSend(pair, { nonce: -1 })
+await result.waitFinalized()
 ```
 
-And that is basic workaround with Phat Contract, it may cover almost all of your use scenarios.
+> [!NOTE]
+> Starting from version 0.5.4, we have introduced a new mode called "send" that handles complex gas fees and cluster balance management.
```

Comment:
Do we have a documentation for send vs tx?

Reply:
I am only mentioned in the README.md and recommended for use. What do you think we can improve here for the comparison?